fontforgeexe/charinfo.c

```diff
@@ -3251,8 +3251,11 @@ return( NULL );
 
     if ( ymax<=ICON_WIDTH ) ymax = ICON_WIDTH;
     if ( ymin>0 ) ymin = 0;
-    if ( xmax<xmin )
+    if ( xmax<xmin ) {
+        free(extras);
```

Comment:
I don't understand what you mean.
Instead of this:
    if ( xmax<xmin )
        return( NULL );
I put this:
    if ( xmax<xmin ) {
        free(extras);
return( NULL );
    }